librz/bin/format/elf/elf_notes.c

```diff
@@ -104,9 +121,15 @@ static RzBinElfPrStatusLayout prstatus_layouts[ARCH_LEN] = {
 	[SPARC_V9] = { SPARC64_REGS_SIZE, SPARC64_PR_STATUS_REG_OFFSET, 64, SPARC64_PR_STATUS_REG_OFFSET_SP },
 	[OPENBSD_SPARC_V9] = { SPARC64_REGS_SIZE, SPARC64_OPENBSD_REG_OFFSET, 64, SPARC64_PR_STATUS_REG_OFFSET_SP },
 
+	[MIPS_32] = { MIPS32_REGS_SIZE, MIPS_GPR32_STATUS_OFFSET, 0, 0 },
```

Comment:
SP offset missing?
It is used actually, so skipping it might not be good.

Reply:
i cannot get a good idea from the GDB sources nor LLVM, so i don't actually know.

Reply:
Ah I see. The SP offset is from the base of the register values.
So in this case the registers `zero, v0, v1, a0-a3..., k0, k1, gp` come before (32bit MIPS I mean).
See how it is used in `Elf_(rz_bin_elf_get_sp_val)`